google-beta/resource_compute_instance_group_manager.go

```diff
@@ -538,50 +469,58 @@ func resourceComputeInstanceGroupManagerUpdate(d *schema.ResourceData, meta inte
 		return err
 	}
 
-	d.Partial(true)
-
-	if _, ok := d.GetOk("rolling_update_policy"); d.Get("update_strategy") == "ROLLING_UPDATE" && !ok {
-		return fmt.Errorf("[rolling_update_policy] must be set when 'update_strategy' is set to 'ROLLING_UPDATE'")
+	updatedManager := &computeBeta.InstanceGroupManager{
+		Fingerprint: d.Get("fingerprint").(string),
 	}
+	var change bool
 
-	// If target_pools changes then update
 	if d.HasChange("target_pools") {
-		targetPools := convertStringSet(d.Get("target_pools").(*schema.Set))
+		updatedManager.TargetPools = convertStringSet(d.Get("target_pools").(*schema.Set))
+		change = true
+	}
 
-		// Build the parameter
-		setTargetPools := &computeBeta.InstanceGroupManagersSetTargetPoolsRequest{
-			Fingerprint: d.Get("fingerprint").(string),
-			TargetPools: targetPools,
+	if d.HasChange("auto_healing_policies") {
+		if v, ok := d.GetOk("auto_healing_policies"); ok {
```

Comment:
If auto_healing_policies changes from set to unset, wouldn't we want to include that change in the request with a forcesend/nullfields?

Reply:
I assume! I'll add a test step for it.